io.openems.backend.timedata.aggregatedinflux/readme.adoc

```diff
@@ -0,0 +1,119 @@
+= Timedata Aggregated InfluxDB 
+
+[NOTE]
+====
+Bundle is needed for highly optimized environments with 
+thousands of connected Edges only. If you do not need that functionality you can skip this chapter. 
+====
+
+OpenEMS Backend implementation for aggregated InfluxDB access.
+
+If many OpenEMS edges are connected to a backend, the increasing number of edges per backend leads to a performance bottleneck.
```

Comment:
`A large number of OpenEMS edges connected to a single backend leads to a performance bottleneck.` would be a better way to write this I think.